e2e/utils/strategy/ManageStrategyDriver.ts

```diff
@@ -0,0 +1,37 @@
+import { CreateStrategyTemplate } from './../../utils/strategy/template';
+import { DebugDriver } from './../../utils/DebugDriver';
+import { navigateTo } from './../../utils/operators';
+import { MyStrategyDriver } from './../../utils/strategy/MyStrategyDriver';
+import { Page } from 'playwright-core';
+
+export class ManageStrategyDriver {
+  constructor(private page: Page) {}
+
+  async createStrategy(config: CreateStrategyTemplate) {
+    const debug = new DebugDriver(this.page);
+    await debug.createStrategy(config);
+    await navigateTo(this.page, '/');
+    const myStrategies = new MyStrategyDriver(this.page);
+    return await myStrategies.getStrategy(1);
+  }
+
+  async waitForEditPage(type: 'deposit' | 'withdraw' | 'renew') {
+    await this.page.waitForURL(`/strategies/edit?type=${type}`, {
+      timeout: 10_000,
+    });
+  }
+
+  async fillBudget(
+    type: 'deposit' | 'withdraw',
+    order: 'buy' | 'sell',
```

Comment:
Nitpick: I think there is a type `Mode` in the CreateStrategyDriver file, maybe we can reuse it. We can do a cleanup of all type later this week though.